Modules/TPC/CMakeLists.txt

```diff
@@ -8,7 +8,8 @@ target_sources(QcTPC PRIVATE src/PID.cxx
                              src/PIDClusterCheck.cxx
                              src/TrackClusterCheck.cxx
                              src/ROCReductor.cxx
-                             src/Clusters.cxx)
+                             src/Clusters.cxx
+                             src/Calibration.cxx)
```

Comment:
No sure how good the name calibration is. Perhaps `CalDetPublisher` or so?

Reply:
I agree, the name is not the best. However, I went with the "naming convention" we had so far in QC. Just name the tasks after what they are looking at. Since the code is in QC it should be clear that this does not calibrate anything. But again, I agree that especially in this case the name is not ideal.

Reply:
Ok, depends on what it will all look at. At the moment it only looks at CalDet Calibration objects. If we want to use it for other stuff as well the name could stay. But I think it would be more natural to have it only look at the CalDet stuff, so Pulser, Pedestal/Noise, Laser Central Electrode.

Reply:
True, I changed it to CalDetPublisher

Reply:
Then I should also change the path in the QCG, right? Or should we leave it at qc/TPC/MO/Calibration?